detectors/boken/requirements.txt

```diff
@@ -2,12 +2,12 @@ numpy==1.18.1
 opencv_python==4.2.0.34
 torch==1.4.0
 torchvision==0.5.0
-boto3==1.17.5
 albumentations==0.5.2
 efficientnet_pytorch==0.7.0
 facenet_pytorch==2.5.1
 Pillow==8.1.1
 pretrainedmodels==0.7.4
 Flask==1.1.2
 pandas==1.0.3
+boto3==1.17.5
```

Comment:
Is this a merge issue? everywhere else  you deleted this dependency.

Reply:
I had removed it from all of the detectors since `app.py` no longer uses it, but boken as a detector still has it as a dependency under `eval_kit` so I needed to add it back for just boken specifically.